free-1.md

```diff
@@ -57,12 +173,30 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I suspect that a lot of time would be dedicated into semantics, specifically
+into calculating the spacing between branches.
+
+The proposed syntax seems very intuitive to me, so maybe there won't be a lot
+to do in terms of language design decisions. 
+But my guess could totally be wrong and maybe there's a lot more research
+required into the language aspects.
+If the language borrows a lot from LaTeX, which seems to make sense for the 
+domain, there would not be much to do in terms of language design.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The project has a very specific set goal, so I think it's not too ambitious.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea because I would be excited to work on this project
+and because I think there's a real need for it.
+
+This would not be a good idea because there may not be a lot of time needed
+in terms of language design.
+Moreover, I could fall on the trap of designing a language that only makes
+sense for me.
```

Comment:
I too have questioned if this is a language mainly useful for you. I think that theres more to it than you. Seeing how ways to turn LaTex into tableau proofs already exists, there clearly is a need for something like this beyond you. Also there are other students who take cs81 and use LaTex!